packages/frontend/core/src/pages/workspace/detail-page/detail-page.tsx

```diff
@@ -318,6 +321,22 @@ export const DetailPage = ({ pageId }: { pageId: string }): ReactElement => {
     }
   }, [jumpOnce, pageRecord]);
 
+  useEffect(() => {
```

Comment:
it seems to me this should be implemented with the new `effect` function (Livedata)

Reply:
No, the new `effect` is used to describe network requests 😂